package.json

```diff
@@ -31,7 +31,7 @@
     "lodash.throttle": "^4.1.1",
     "markdown-it": "^13.0.1",
     "marked": "^4.0.10",
-    "monaco-editor-core": "^0.39.0",
+    "monaco-editor": "~0.40.0",
```

Comment:
Near as I can tell, `@monaco-editor/react` depends on `monaco-editor`, not `-core`, which means that using it should both give us more faithful typing and also doesn't require bringing in a separate library.